### DISCUSSION TITLE
New Error About Leading Trailing Whitespace

### DISCUSSION THREAD

**Discussion by NuclearPowerNerd** [dendronhq/dendron#3405] [Q&A] [answered]
Hello,

I've had a lot of great success using Dendron as my personal note taking tool. I have just come across my first error though and I do not know how to diagnose it. I noticed today one of my notes will now no longer open if I use my usual method which is pressing `ctrl+L` in VSCode, using tab to navigate my note hierarchy, then hitting `enter` when I find the one I want to open. On this particular note I get this error that I've never seen before nor know how to fix (I've opened this note successfully the in the past, as recently as August 1st). 

`Hierarchies cannot have leading / trailing whitespace or be empty.`

The full note path is `Fluids.Navier Stokes. Two Dimensional`

I can open the note if I manually find it in the vault and then preview it or if I navigate to it in the graph panel. But as I said my usual method is not working. 

Is the error occurring because of the blank space in front of `Two`? That'd be weird because as I said I have successfully opened this note in the past and not had this error and I've not changed the name. If I try to change the name using the `Dendron: Rename Note` command I get the same error as above.

I would appreciate any advice on how to resolve this.

**Comment by hikchoi** [comment, marked as answer]
Yes, this is because of the leading whitespace in front of `Two`. This validation has been introduced recently.

For the time being, you can manually rename the note (with the regular `rename` that VSCode provides) so that it becomes `Fluids.Navier Stokes.Two Dimensional.md` and lookup will work again.

We'll add a way to automatically fix invalid filenames so that this doesn't happen again.
Sorry about the inconvenience.

**Reply by NuclearPowerNerd** [reply]
Thanks for the quick reply! 👍

**Reply by hikchoi** [reply]
@NuclearPowerNerd A fix has been pushed with version 0.110.0. Now you will be able to look up existing notes even if they have invalid file names.
